mne/report.py

```diff
@@ -839,6 +840,51 @@ def add_images_to_section(self, fnames, captions, scale=None,
             self._sectionlabels.append(sectionvar)
             self.html.append(html)
 
+    def add_htmls_to_section(self, htmls, captions, section):
+        """Append htmls to the report.
+
+        Parameters
+        ----------
+        htmls : list of str
+            A list of html str.
+        captions : list of str
+            A list of captions to the htmls.
+        section : str
+            Name of the section. If section already exists, the images
+            will be appended to the end of the section.
+        """
+        htmls, captions = self._validate_input(htmls, captions, section)
+        for html, caption in zip(htmls, captions):
+            caption = 'custom plot' if caption == '' else caption
+            sectionvar = self._sectionvars[section]
+
+            self.fnames.append('%s-#-%s-#-custom' % (caption, sectionvar))
+            self._sectionlabels.append(sectionvar)
+            self.html.append(html)
+
+    def render_bem(self, subject, subjects_dir, decim, n_jobs):
+        """Render an bem slider html str.
+
+        Parameters
+        ----------
+        subject : str
+            Subject name.
+        subjects_dir : str | None
+            Path to the SUBJECTS_DIR. If None, the path is obtained by using
+            the environment variable SUBJECTS_DIR.
```

Comment:
`subjects_dir` is usually moved toward the end since it's less likely to be necessary / useful than other arguments (e.g., `decim`)